src/traits.rs

```diff
@@ -29,16 +29,16 @@ pub trait Instantiate {
     /// ## Returns
     /// Returns a Response containing the messages to instantiate the token.
     ///
-    /// ## Example
-    /// ```
+    /// ## Example (pseudocode)
+    /// ```ignore
```

Comment:
@apollo-pablo can you verify that this works with the test framwork before we merge?

Reply:
I cannot verify it since this dependency is not used by the testing framework directly, but used by the contracts. But running `cargo test` on the branch work, so it is expected that this fix the problem.

Reply:
perfect! sounds good